app/assets/stylesheets/application.css.scss

```diff
@@ -17,4 +17,5 @@
  */
 body {
   padding-top: 50px;
+  background-color: #fafafa;
```

Comment:
Color literals like `#fafafa` should only be used in variable declarations; they should be referred to via variable everywhere else.